core/src/peer_record.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto2";
```

Comment:
Could you replace this file with https://github.com/libp2p/go-libp2p-core/blob/master/peer/pb/peer_record.proto ?